implementations/rust/ockam/ockam_app_lib/src/enroll/enroll_user.rs

```diff
@@ -180,8 +178,11 @@ impl AppState {
             None => {
                 self.notify(Notification {
                     kind: Kind::Information,
-                    title: "Creating a new project...".to_string(),
-                    message: "This might take a few minutes".to_string(),
+                    title: "Provisioning a project".to_string(),
+                    message:
+                        "We're provisioning a dedicated project for you in Ockam Orchestrator.\
```

Comment:
There is a space missing at the end of `Ockam Orchestrator.`. It should be `Ockam Orchestrator. \`